### ISSUE TITLE
compiling swarm on osx 1.8.5

### ISSUE THREAD

**Issue by colinbrislawn** [torognes/swarm#69]
When building swarm from source, I receive this error. This is an older mac, so it could also be a hardware limitation. 

```
cd src/
make
g++ -Wall -Wsign-compare -O3 -msse2 -mtune=core2 -Icityhash -g -c -o swarm.o swarm.cc
In file included from swarm.cc:24:
swarm.h:28:23: error: x86intrin.h: No such file or directory
make: *** [swarm.o] Error 1
```

**Comment by frederic-mahe** [comment]
Hi @colinbrislawn,

it could be a problem with the compiler. A similar issue was reported for vsearch ([issue 146](https://github.com/torognes/vsearch/issues/146)). I think @torognes found a solution.

Thanks for reporting that issue.

**Comment by torognes** [comment]
This problem should be fixed in version 2.1.6.
